packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -58,7 +57,8 @@ export async function createNewFluidFile(
     }
 
     const filePath = newFileInfo.filePath ? encodeURIComponent(`/${newFileInfo.filePath}`) : "";
-    const encodedFilename = encodeURIComponent(`${newFileInfo.filename}.fluid`);
+    const fileName = newFileInfo.filename.indexOf(".") === -1 ? `${newFileInfo.filename}.fluid` : newFileInfo.filename;
```

Comment:
You might want to search the end of string only, not any occurrence in string

Reply:
Sorry this was an old change. Rmeoved all this.